src/DependencyInjection/Configuration.php

```diff
@@ -164,13 +164,17 @@ private function getOptionsConfig(): ArrayNodeDefinition
                 ->scalarNode('security_cover_url')
                     ->defaultValue('https://placehold.co/1000')
                 ->end()
-                ->scalarNode('theme_base')
-                    ->defaultValue('slate')
+                ->enumNode('theme_base')
+                    ->values(['slate', 'gray', 'zinc', 'neutral', 'stone'])
+                    ->defaultValue('gray')
                 ->end()
-                ->scalarNode('theme_radius')
-                    ->defaultValue('0.5')
+                ->floatNode('theme_radius')
+                    ->min(0)
+                    ->max(2)
```

Comment:
I was really close to force choices like `0, 0.5, 1, 1.5, 2` and not a simple min max, which mean we can do `0.8`.
But that mean, we also need to check that in the ContextHelper setter.
WDYT @kevinpapst?